ui/app/src/styles/async-video-player.scss

```diff
@@ -0,0 +1,28 @@
+.async-video-player--unavailable-message, .async-video-player--loading {
```

Comment:
Check for reuse of videojs's classes/markup instead of duplicating the styles. Code only your style extensions/additions.